wiki/Beatmap_Editor/Song_Setup/fr.md

```diff
@@ -1,193 +1,259 @@
 
-Aussi connu en tant que "**HP**" lors de la sélection d'une beatmap.
+*Notice: Le mode de jeu spécifique [Ranking Criterias](/wiki/Ranking_Criteria) ont des valeurs de classement qui devraient être utilisées.*
 
-La vitesse à laquelle la vie diminue, qui est toujours constante sauf pendant une pause. Plus de vie est perdue avec des miss. La vie peut être récupérée en obtenant des [valeurs de points](/wiki/Score) et en tournant des spinners. Plus de vie est récupérée en obtenant [des Geki et des Katu](/wiki/Score) à la fin d'un combo.
+Les réglages de difficulté de la carte de rythme ; chaque réglage affecte différents aspects de la carte de rythme.
 
-### Circle Size (Taille des cercles; osu!, osu!catch, osu!taiko seulement)
+*Remarque : En maintenant la touche `Shift` enfoncée, on peut régler avec précision la valeur 0,1 par rapport au réglage standard de la valeur 1.
 
-![Aperçu des différentes tailles de cercle](img/Circle_sizes.png "Aperçu des différentes tailles de cercle")
+La notation courante pour représenter un réglage de difficulté et sa valeur est l'abréviation du réglage suivie de la valeur du réglage. Par exemple, `CS4` et `HP4.3` seraient une carte de rythme avec une taille de cercle de 4 et un taux de drainage de HP de 4,3 respectivement. En théorie, les réglages de difficulté négative peuvent ressembler à `AR-2` pour un taux d'approche de -2.
 
-![Graphique d'AR détaillé](/wiki/shared/ARTable.jpg "Graphique d'AR détaillé")
+### Débit de drainage HP
 
-Aussi connu en tant que "**CS**" lors de la sélection d'une beatmap.
+Le taux de drainage HP, communément abrégé en HP, détermine le taux de récupération de la santé à partir de bons scores de coups et la punition de la santé à partir de mauvais scores de coups. La valeur varie de 0 à 10 ; des valeurs de HP plus élevées donneront une récupération de santé plus faible avec une punition de santé plus élevée.
 
-Ceci détermine la taille des cercles et des sliders. La valeur varie entre 2 et 7 avec les plus petites valeurs donnant des plus grosses tailles. Les petits cercles rendent la map difficile en exigeant le joueur à être plus précis.
+Dans osu!standard et osu!catch, cela affecte également le [Passive HP](/wiki/Passive_HP) (la vitesse constante à laquelle HP diminue, sauf pendant les[Breaks](/wiki/Breaks).
 
-La plupart des maps utilisent les tailles 3, 4 et 5. 6 est parfois utilisé et 7 est quasiment jamais utilisé (les objets font quasiment la taille du curseur par défaut). 7 a plus de chances d'être utilisé un utilisant le mod [Hard Rock](/wiki/Game_Modifiers). Un exemple d'une [beatmap utilisant 3, 5, 6, et 7 par défaut](https://osu.ppy.sh/beatmapsets/2008) et une [beatmap qui utilise 0, 1, 3 et 4 par défaut](https://osu.ppy.sh/beatmapsets/1208).
+*Note : Plus de HP est récupéré avec [Geki and Katu](/wiki/Score/#osu) les scores de coups, qui peuvent être donnés à partir du dernier objet touché dans un combo.*
```

Comment:
line 73 has `Note` translated, but this line doesn't?

This happens throughout the article.